contrib/kubernetes/manifests/registry-s3-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-keyfile
+type: Opaque
+data:
+  accesskey: {base64 encode of the access key}
+  secretkey: {base64 encode of the secret key}
+  region: {base64 encode of the region name}
+  bucket: {base64 encode of the buket name}
```

Comment:
shouldn't Region and bucker  be a part of RC rather than secret , setting them as envs